src/mono/mono/btls/btls-error.c

```diff
@@ -58,8 +58,8 @@ mono_btls_error_get_reason (int error)
         case SSL_R_NO_RENEGOTIATION:
             return 100;
         default:
-            return 0;
+            break;
     }
```

Comment:
Since this is a change in behavior, a plausible reason for the change needs to be provided, together with tests.  And please do not hide such things in seemingly "cosmetic" PR's.

This patch changes to return value from 0 to the return value of `ERR_GET_REASON (error)` - without giving any explanation for this change.